src/containerapp/azext_containerapp/_clients.py

```diff
@@ -876,7 +900,7 @@ class ContainerAppsJobClient():
     api_version = CURRENT_API_VERSION
 
     @classmethod
-    def create_or_update(cls, cmd, resource_group_name, name, containerapp_job_envelope, no_wait=False):
+    def create_or_update(cls, cmd, resource_group_name, name, containerapp_job_envelope, no_wait=False, show_sensitive_values=False):
```

Comment:
Please add this logic to `ContainerAppPreviewClient`. I will remove All clients which has been move to azure-cli such as `ContainerAppClient`.